# Panzootic chytrid fungus exploits diverse amphibian host environments through plastic infection strategies

### Abstract of the research article

While some pathogens are limited to single species, others can colonize many hosts, likely contributing to the emergence of novel disease outbreaks. Despite this biodiversity threat, traits associated with host niche expansions are not well understood in multi-host pathogens. Here, we aimed to uncover functional machinery driving multi-host invasion by focusing on *Batrachochytrium dendrobatidis* (*Bd*), a pathogen that infects the skin of hundreds of amphibians worldwide. We performed a meta-analysis of *Bd* gene expression using data from published infection experiments and newly generated profiles. We analyzed *Bd* transcriptomic landscapes across the skin of 14 host species, reconstructed *Bd* isolates phylogenetic relationships, and inferred the origin and evolutionary history of differentially expressed genes under a phylogenetic framework comprising other 12 zoosporic fungi. *Bd* displayed plastic infection strategies when challenged by hosts with different disease susceptibility. Our analyses identified sets of differentially expressed genes under host environments with similar infection outcome. We stressed nutritional immunity and gene silencing as important processes required to overcome challenging skin environments in less susceptible hosts. Overall, *Bd* genes expressed during amphibian skin exploitation have arisen mainly via gene duplications with great family expansions, increasing the gene copy events previously described for this fungal species. Finally, we provide a comprehensive gene dataset that can be used to further examine eco-evolutionary hypotheses for this host-pathogen system. Our study supports the idea that host environments exert contrasting selective pressures, such that gene expression plasticity could be one of the evolutionary keys leading to the success of multi-host pathogens.

### Data information

The excel file (Torres-Sanchez_et_al_Bd_gene_expression_across_hosts_FileS1.xlsx) is composed by five sheets containing raw read counts for the 137 samples (sheet 1: raw_counts) and all Bd genes with their annotations, differential gene expression results, and gene family information (sheet 2: DEG_genes), and results from PANTHER overrepresentation test (sheet 3: PANTHER_Invasion, 4: PANTHER_Virulence, and 5: PANTHER_Defense_transmission). Raw counts are displayed in a table manner with Bd counts per gene (8700 genes included in the Bd JAM81 representative genome, GCF_000203795.1) in each row for the 137 samples with their SRA codes in each column. For the differentially expressed genes, each gene annotations include NCBI accession numbers (NCBI_accession_numbers), Uniport best hit information (Uniprot_id and Uniprot_protein_name), Pfam best hit information (Pfam_id and Pfam_description), SignalP best hit category (SignalP), and Gene Ontology domain description (GO_biological_process, GO_cellular_component, and GO_molecular_function). In this second excel sheet, we included information from the differential expression analyses: mean expression across samples (BaseMean_expression); fold change in logarithmic scale and adjust p-values for the contrasts susceptible host versus in culture (log2FC_Susceptible and padj_Susceptible), partially susceptible host versus in culture (log2FC_Partial_Susceptible and padj_Partial_susceptible), and non-susceptible hosts versus in culture (log2FC_Non-Susceptible and padj_Non-Susceptible); classification from the contrast zoospore versus sporangia (Life_stage_signature); and gene category based on the decision tree represented in Fig. 1B (Potential_gene_category). Additionally, we reported information of the gene family (Orthogroup_id) including the number of both Bd and total genes in each gene family (Number_Bd_genes and Total_gene_number, respectively), and also number of species (Species_member_number). Finally, we included duplication information through the gene family categorization based on origin and evolutionary history as well as GC content and the number of nucleotides (Orthogroup_member_category, Orthogroup_Bd_duplication_category, gene_length, and gene_%GC). 

The txt file (Torres-Sanchez_et_al_Bd_gene_expression_across_hosts_FileS2.txt) contains complete code for all analyses presented in the article.


### Citation

When using the data or the code please cite the following paper:  
Torres-SÃ¡nchez et al. (2022). Panzootic chytrid fungus exploits diverse amphibian host environments through plastic infection strategies. Molecular Ecology. https://onlinelibrary.wiley.com/doi/10.1111/mec.16601


